### ISSUE TITLE
请问fully fine-tune有遇到过overflow的问题吗？

### ISSUE THREAD

**Issue by mynewstart** [mark1879/Baichuan-13B-Finetuning#5]

**Comment by larsoncs** [comment]
我也遇到这个问题了，“Exception: Current loss scale already at minimum - cannot decrease scale”

**Comment by mark1879** [comment]
全量微调的话，可以去 https://github.com/hiyouga/LLaMA-Efficient-Tuning 看一下

**Comment by mynewstart** [comment]
tokenizer训练的时候不要设为left
